### PR TITLE
Convert to Swift 3 and added a simple dummy test

### PR DESCRIPTION
**Pull request by ezefranca**
I was going to add cocoapods but it would be lame :)

### CODE REVIEW

File: Generator/FCBBarCodeGenerator.swift [hunk 1]
```diff
@@ -15,7 +15,7 @@ struct FCBBarCodeGenerator {
     
     // MARK: Public Methods
     
-    func barcodeWithCode(code: String, type: FCBBarcodeType, size: CGSize) -> UIImage? {
+    func barcodeWithCode(_ code: String, type: FCBBarcodeType, size: CGSize) -> UIImage? {
```
Comment (Bunn) [inline Generator/FCBBarCodeGenerator.swift]: @ezefranca would't be more "swiftier" if this method name would be changed to
`func barcode(with code: String, type: FCBBarcodeType, size: CGSize) -> UIImage?`

File: Generator/FCBBarCodeGenerator.swift [hunk 2]
```diff
@@ -26,25 +26,25 @@ struct FCBBarCodeGenerator {
     
     // MARK: Private Methods
     
-    private func imageWithFilter(filter : CIFilter, size: CGSize) -> UIImage? {
+    fileprivate func imageWithFilter(_ filter : CIFilter, size: CGSize) -> UIImage? {
```
Comment (Bunn) [inline Generator/FCBBarCodeGenerator.swift]: @ezefranca same thing here, maybe
`fileprivate func image(with filter : CIFilter, size: CGSize) -> UIImage?`
would look better on swift 3, what do you think?

File: Generator/FCBBarCodeGenerator.swift [hunk 3]
```diff
@@ -26,25 +26,25 @@ struct FCBBarCodeGenerator {
...
+            let transformedImage = image.applying(CGAffineTransform(scaleX: scaleX, y: scaleY))
             
-            return UIImage(CIImage: transformedImage)
+            return UIImage(ciImage: transformedImage)
         }
         return nil
     }
     
-    private func filterWithCode(code: String, type: FCBBarcodeType) -> CIFilter? {
+    fileprivate func filterWithCode(_ code: String, type: FCBBarcodeType) -> CIFilter? {
```
Comment (Bunn) [inline Generator/FCBBarCodeGenerator.swift]: @ezefranca same thing here,
`fileprivate func filter(with code: String, type: FCBBarcodeType) -> CIFilter?`

**Comment by Bunn** [comment]
@ezefranca thanks for this :) There are some minor changes that I think it would look better, but let me know what you think.

**Review by Bunn — CHANGES_REQUESTED** [review]
@ezefranca thanks for this :) There are some minor changes that I think it would look better, but let me know what you think.

File: Generator/FCBBarCodeGenerator.swift (hunk 3 above)
Comment (ezefranca) [reply Generator/FCBBarCodeGenerator.swift]: Better!!

File: Generator/FCBBarCodeGenerator.swift (hunk 2 above)
Comment (ezefranca) [reply Generator/FCBBarCodeGenerator.swift]: 🎉

File: Generator/FCBBarCodeGenerator.swift (hunk 1 above)
Comment (ezefranca) [reply Generator/FCBBarCodeGenerator.swift]: 👍

**Comment by ezefranca** [comment]
Nice @Bunn , Sometimes Obj-C still alive on me hahahaha

**Comment by Bunn** [comment]
@ezefranca thanks, LGTM but you forgot to update the tests haha.

**Comment by ezefranca** [comment]
@Bunn I'm testing you, now I think everything is ok. 

Kids, learn from me and repeat with me: 
- I will never push without run the tests

[]'s

**Pull request merged**